src/testing/jest/install-dependencies.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+set -e
+
+# Get the directory where this script is located
+SCRIPT_DIR=$(cd "$(dirname "$0")" && pwd)
+
+# Change to the directory where the script is located so that it can be called from anywhere
+cd "$SCRIPT_DIR"
+
+# Flag to check if any jest-[SOME_NUMBER] directories were found and processed
+found=false
+
+# Loop through directories start with 'jest-', e.g. `jest-27-and-under`, `jest-28`, etc.
+for dir in jest-*; do
+    if [[ -d "$dir" ]]; then
+        found=true
+
+        if cd "$dir"; then
+            echo "Installing dependencies in $dir"
+            npm ci || {
+                echo "Error: npm ci failed in directory $dir"
+                exit 1
+            }
+            echo "" # print a newline to separate sequential installs
+        else
+            echo "Error: Cannot enter directory $dir"
+            exit 1
+        fi
+
+        # go back to where we started
+        cd - > /dev/null || {
+            echo "Error: Unable to leave $dir"
+            exit 1
+        }
```

Comment:
with `set -e` won't we exit for a non-zero exit code here if the `cd` fails anyhow?

Reply:
Oh, good point (I missed this comment when I resubmitted for review). Let me double check on that. We may be able to get rid of this + another `npm ci` related failure check